alodataset/transforms.py

```diff
@@ -881,8 +881,8 @@ def apply(self, frame: Frame, center: Union[Tuple[int, int], Tuple[float, float]
                 Default: (0.5, 0.5)
         """
         if isinstance(center[0], float):
-            center_x = int(frame.W * center[0])
-            center_y = int(frame.H * center[1])
+            center_x = int(frame.W * center[0]) - 1
```

Comment:
I don't think that shifting the center is good choice. with the same example above you still have the same problem.
```python
ValueError: Crop coordinates out of image border. Image size: (50, 20), Crop coordinate (top, left, bot, right): (-1, 14, 19, 34)
```